pyproject.toml

```diff
@@ -12,11 +12,7 @@ authors = [{ name = 'Alexander Renz', email = 'alex.renz@gmx.net' }]
 license = { text = "MIT" }
 readme = "README.md"
 requires-python = '<3.12'
-dependencies = ["geopandas", "matplotlib<3.8", "numpy", "pandas", "shapely"]
-
-[project.optional-dependencies]
```

Comment:
@HeiJuli You accidentally deleted some lines …

Reply:
thanks for viewing the mistake!
it should be fixed

Reply:
You have moved the line. The dependencies need to be at line 15!